index.html

```diff
@@ -1,11 +1,153 @@
+    b = +prompt("Введите целочисленное значение B");
+    if (a >= 0 && b >= 0) {
+        c = a - b; console.log(c);
+    } else if (a < 0 && b < 0) {
+        c = a * b; console.log(c);
+    } else {
+        c = a + b; console.log(c);
+    }
+
+    // Задание 4
+    a = +prompt("Введите число от 0 до 15 включительно");
+    while (a < 0 || a >  15) {
+        a = +prompt("Неправильное значение. Введите число от 0 до 15 включительно")
+    }
+    switch (a) {
+        case 0:
+            console.log("0, 1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13, 14, 15");
+            break;
+        case 1:
+            console.log("1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13, 14, 15");
+            break;
+        case 2:
+            console.log("2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13, 14, 15");
+            break;
+        case 3:
+            console.log("3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13, 14, 15");
+            break;
+        case 4:
+            console.log("4, 5, 6, 7, 8, 9, 10, 11, 12, 13, 14, 15");
+            break;
+        case 5:
+            console.log("5, 6, 7, 8, 9, 10, 11, 12, 13, 14, 15");
+            break;
+        case 6:
+            console.log("6, 7, 8, 9, 10, 11, 12, 13, 14, 15");
+            break;
+        case 7:
+            console.log("7, 8, 9, 10, 11, 12, 13, 14, 15");
+            break;
+        case 8:
+            console.log("8, 9, 10, 11, 12, 13, 14, 15");
+            break;
+        case 9:
+            console.log("9, 10, 11, 12, 13, 14, 15");
+            break;
+        case 10:
+            console.log("10, 11, 12, 13, 14, 15");
+            break;
+        case 11:
+            console.log("11, 12, 13, 14, 15");
+            break;
+        case 12:
+            console.log("12, 13, 14, 15");
+            break;
+        case 13:
+            console.log("13, 14, 15");
+            break;
+        case 14:
+            console.log("14, 15");
+            break;
+        case 15:
+            console.log("15");
+            break;
+    }
+
+
+    // Задание 5
+    var result;
+    function sum(arg1, arg2) {
+        result = arg1 + arg2;
+        return result;
+    }
+    function diff(arg1, arg2) {
+        result = arg1 - arg2;
+        return result;
+    }
+    function increase(arg1, arg2) {
+        result = arg1 * arg2;
+        return result;
+    }
+    function division(arg1, arg2) {
+        result = arg1 / arg2;
+        return result;
+    }
+
+    // Задание 6
+    function mathOperation(arg1, arg2, operation) {
+        switch (operation) {
+            case sum:
+                console.log(sum(arg1, arg2));
+                break;
+            case diff:
+                console.log(diff(arg1, arg2));
+                break;
+            case increase:
+                console.log(increase(arg1, arg2));
+                break;
+            case division:
+                console.log(division(arg1, arg2));
+                break;
+        }
+    }
+    mathOperation(55, 5, sum);
```

Comment:
А вот здесь в свою очередь уже можно использовать console.log:
```suggestion
    console.log(mathOperation(55, 5, sum));
```